Example/ios/Podfile.lock

```diff
@@ -312,9 +314,8 @@ PODS:
     - React-RCTVibration
     - ReactCommon/turbomodule/core
     - Yoga
-  - RNScreens (3.4.0):
+  - RNScreens (2.17.1):
```

Comment:
Why do we downgrade screens here?

Reply:
I only run standard `npx pod-install` and on that branch in example's `package.json` there is downgraded version of screens (which I didn't touch)